Samples/macOS-Swift/Shared/Base.lproj/Main.storyboard

```diff
@@ -852,16 +852,6 @@
                                             <action selector="diskWriteException:" target="XfG-lQ-9wD" id="3dp-EG-XYM"/>
                                         </connections>
                                     </button>
-                                    <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="UAi-Ph-r0g" userLabel="asyncCrash">
```

Comment:
`h`: Why was this removed?

Reply:
it didn't actually do anything, but i can remove it in its own PR instead of rolling it into these changes

Reply:
Thank you, that will make it easier to understand in the future